examples/tutorial/basic-1/programs/basic-1/src/lib.rs

```diff
@@ -22,7 +22,7 @@ mod basic_1 {
 #[derive(Accounts)]
 pub struct Initialize<'info> {
     #[account(init, payer = user, space = 8 + 8)]
-    pub my_account: Account<'info, MyAccount>,
+    pub my_account: ProgramAccount<'info, MyAccount>,
```

Comment:
`Account` is preferred. The docs should be updated instead of the code.

Reply:
Ah, OK. :) :+1: